README.md

```diff
@@ -75,21 +69,19 @@ The easies way to generate all the required files is to use [fastlane][1] built
 
 After the push certificate and the keys are generated, please submit *pem*, *p12* and *pkey* files to support@salemove.com.
 
-### Push Notifications
-
-In order for the project to receive chat messages and other events, [push capabilities][3] should be enabled for the target.
-
 ### Project Configuration
 
 The SalemoveSDK push service relies on [environment variables][4] so please make sure to add the `DEBUG` `1` to the run configuration. This will differentiate the gateway used to submit the messages.
 
+The SalemoveSDK is using the audio/video capabilities of the device so the hosting app needs [NSCameraUsageDescription] [11] and [NSMicrophoneUsageDescription] [12] added to the plist file.
```

Comment:
The SalemoveSDK uses the audio/video capabilities of the device. As such, the hosting app needs